src/files-and-videos/files-page/FilesPage.test.jsx

```diff
@@ -43,6 +43,9 @@ import messages from '../generic/messages';
 import filesPageMessages from './messages';
 import { updateFileValues } from './data/utils';
 
+// eslint-disable-next-line no-promise-executor-return
+const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
```

Comment:
These tests were definitely running (unlike [these](https://github.com/openedx/frontend-app-course-authoring/pull/1052/commits/af509998b96f2129731ad09a077605edbb8058ed), which weren't). I don't know how they were passing before.

I'll [try to find a better way](https://github.com/openedx/frontend-app-course-authoring/pull/1062#discussion_r1626348053) to handle this instead of `sleep`.

Reply:
See [3f53e96](https://github.com/openedx/frontend-app-course-authoring/pull/1052/commits/3f53e96268bd7e5090c48238f0fae642c0eb3802).

Reply:
I like this fix better than the sleep, its much cleaner!